release.json

```diff
@@ -47,8 +47,8 @@
         "INTEGRATIONS_CORE_VERSION": "7.56.0-rc.2",
         "OMNIBUS_SOFTWARE_VERSION": "7.56.0-rc.3",
         "OMNIBUS_RUBY_VERSION": "7.56.0-rc.1",
-        "JMXFETCH_VERSION": "0.49.5",
-        "JMXFETCH_HASH": "d29f945af0353905a5c801ab6b6ac027aa0cb27535470f464ca9254f6f36ef23",
+        "JMXFETCH_VERSION": "0.49.6",
```

Comment:
Shall we backport this to the `agent6` release branch?

Reply:
It doesn't have any security updates so I don't think we need to. I'm not 100% sure the current guidelines on when/what should be back ported to Agent 6 or not.